webapp/sources/api-doc/paths/archives/import.yml

```diff
@@ -14,6 +14,12 @@ post:
               type: string
               format: binary
               description: The ZIP archive file containing policies in a conventional layout and serialization format
+            merge:
+              type: string
+              description: Optionnal merge option of the import. Default `override-all`. `keep-rule-groups` will keep existing target definition for existing rules.
```

Comment:
does that means that "keep-rule-group" won't synchronize the group, or that t will merge (resulting in targets being existing group + group from the import )?

Reply:
If you import a rule that already exists, it will not change the current target of the rule (ignore the target of the imported rules)

Reply:
updated the doc to specify that archive values are ignored in the seconde case.